src/pages/downloads.tsx

```diff
@@ -195,36 +199,42 @@ export const getStaticProps: GetStaticProps = async () => {
           // linux
           ALL_LINUX_STABLE_RELEASES: getSortedReleases(
             LINUX_STABLE_RELEASES_DATA,
-            LINUX_ALLTOOLS_STABLE_RELEASES_DATA
+            LINUX_ALLTOOLS_STABLE_RELEASES_DATA,
+            AMOUNT_OF_RELEASES_PER_LINUX
           ),
           ALL_LINUX_DEV_BUILDS: getSortedReleases(
             LINUX_DEV_BUILDS_DATA,
-            LINUX_ALLTOOLS_DEV_BUILDS_DATA
+            LINUX_ALLTOOLS_DEV_BUILDS_DATA,
+            AMOUNT_OF_RELEASES_PER_LINUX
           ),
           // macOS
           ALL_MACOS_STABLE_RELEASES: getSortedReleases(
             MACOS_STABLE_RELEASES_DATA,
-            MACOS_ALLTOOLS_STABLE_RELEASES_DATA
+            MACOS_ALLTOOLS_STABLE_RELEASES_DATA,
+            AMOUNT_OF_RELEASES_PER_DARWIN
           ),
           ALL_MACOS_DEV_BUILDS: getSortedReleases(
             MACOS_DEV_BUILDS_DATA,
-            MACOS_ALLTOOLS_DEV_BUILDS_DATA
+            MACOS_ALLTOOLS_DEV_BUILDS_DATA,
+            AMOUNT_OF_RELEASES_PER_DARWIN
           ),
           // windows
           ALL_WINDOWS_STABLE_RELEASES: getSortedReleases(
             WINDOWS_STABLE_RELEASES_DATA,
-            WINDOWS_ALLTOOLS_STABLE_RELEASES_DATA
+            WINDOWS_ALLTOOLS_STABLE_RELEASES_DATA,
+            AMOUNT_OF_RELEASES_PER_WINDOWS
           ),
           ALL_WINDOWS_DEV_BUILDS: getSortedReleases(
             WINDOWS_DEV_BUILDS_DATA,
-            WINDOWS_ALLTOOLS_DEV_BUILDS_DATA
+            WINDOWS_ALLTOOLS_DEV_BUILDS_DATA,
+            AMOUNT_OF_RELEASES_PER_WINDOWS
           ),
           // android
-          ALL_ANDROID_STABLE_RELEASES: getSortedReleases(ANDROID_STABLE_RELEASES_DATA),
-          ALL_ANDROID_DEV_BUILDS: getSortedReleases(ANDROID_DEV_BUILDS_DATA),
+          ALL_ANDROID_STABLE_RELEASES: getSortedReleases(ANDROID_STABLE_RELEASES_DATA, [], AMOUNT_OF_RELEASES_PER_MOBILE),
```

Comment:
why the empty `[]` array for android and ios releases?

Reply:
Function takes 3 params, and one is an array, so passing empty array.